blockstore/splitstore/README.md

```diff
@@ -3,129 +3,111 @@
 The SplitStore was first introduced in lotus v1.5.1, as an experiment
 in reducing the performance impact of large blockstores.
 
-With lotus v1.11.1, we introduce the next iteration in design and
-implementation, which we call SplitStore v1.
-
-The new design (see [#6474](https://github.com/filecoin-project/lotus/pull/6474)
-evolves the splitstore to be a freestanding compacting blockstore that
-allows us to keep a small (60-100GB) working set in a hot blockstore
-and reliably archive out of scope objects in a coldstore.  The
-coldstore can also be a discard store, whereby out of scope objects
-are discarded or a regular badger blockstore (the default), which can
-be periodically garbage collected according to configurable user
-retention policies.
+With lotus v1.19.0, we introduce the next iteration in design and
+implementation, which we call SplitStore v2.
+
+The new design (see [#9056](https://github.com/filecoin-project/lotus/pull/9056) and [#9128](https://github.com/filecoin-project/lotus/discussions/9128))
+evolves the splitstore to be a freestanding compacting blockstore that allows you to keep a small 60 GiB to 275 GiB working set in a hot blockstore and reliably archive out-of-scope objects in a coldstore. The coldstore can also be a `discard` store, whereby out-of-scope objects are discarded, a `universal` store, which will store all chain data or a `messages` store which will only store on-chain messages. The `messaqges` badger blockstore is the default storage type.
 
 To enable the splitstore, edit `.lotus/config.toml` and add the following:
 ```
 [Chainstore]
+  # type: bool
+  # env var: LOTUS_CHAINSTORE_ENABLESPLITSTORE
   EnableSplitstore = true
 ```
 
-If you intend to use the discard coldstore, your also need to add the following:
+If you intend to use the `discard` coldstore, your also need to add the following:
 ```
-  [Chainstore.Splitstore]
-    ColdStoreType = "discard"
+[Chainstore.Splitstore]
+  # ColdStoreType specifies the type of the coldstore.
+  # It can be "messages" (default) to store only messages, "universal" to store all chain state or "discard" for discarding cold blocks.
+  #
+  # type: string
+  # env var: LOTUS_CHAINSTORE_SPLITSTORE_COLDSTORETYPE
+  ColdStoreType = "discard"
 ```
-In general you _should not_ have to use the discard store, unless you
-are running a network assistive node (like a bootstrapper or booster)
-or have very constrained hardware with not enough disk space to
-maintain a coldstore, even with garbage collection. It is also appropriate
-for small nodes that are simply watching the chain.
-
-*Warning:* Using the discard store for a general purpose node is discouraged, unless
-you really know what you are doing. Use it at your own risk.
-
 ## Configuration Options
 
-These are options in the `[Chainstore.Splitstore]` section of the configuration:
-
-- `HotStoreType` -- specifies the type of hotstore to use.
```

Comment:
I think only the ColdStoreType config value is out of date.  All the rest are more thorough than the config comments and still relevant so if they are still relevant I'd support keeping them around.